<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>role</class>
 <widget class="QMainWindow" name="role">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F5DEB3;
border:8px solid;
border-radius:5px;
border-color:#A0522D;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Sitka Subheading&quot;;
border-image: url(:/label.png);
padding-bottom:2px;
color:#000;
</string>
    </property>
    <property name="text">
     <string>Choose Server or Client</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="server_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#4169E1;
}
QPushButton:hover{
font: 14pt &quot;MS Reference Sans Serif&quot;;
}

</string>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="client_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#FFA500;
}
QPushButton:hover{
font: 14pt &quot;MS Reference Sans Serif&quot;;
}

</string>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_server">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#000;
font: 11pt &quot;MS Serif&quot;;
font-weight:800;</string>
    </property>
    <property name="text">
     <string>Server IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="getip_edt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding-left:8px;
border-radius:10px;
color:#00000;
background-color:#FFF8DC;
font-size:17px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter server IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="setip_lbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding-left:8px;
border-radius:10px;
color:#696969;
background-color:#FFF8DC;
font-size:17px;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_btn">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>290</y>
      <width>290</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#2E8B57;
}
QPushButton:hover{
background-color:#3CB371;
}

</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Sitka Banner Semibold&quot;;
color:#FF0000;</string>
    </property>
    <property name="text">
     <string>Not Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wait_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>301</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#4682B4;
}
QPushButton:hover{
background-color:#6495ED;
}

</string>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="showUsername_lbl">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Sitka Banner Semibold&quot;;
color:#696969;</string>
    </property>
    <property name="text">
     <string>Player not ready!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="go_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>301</width>
      <height>50</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-radius:20px;
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
background-color:#2E8B57;
}
QPushButton:hover{
background-color:#3CB371;
}

</string>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
